src/FT81x.h

```diff
@@ -377,6 +377,20 @@ class FT81x {
     */
     void drawRect(const int16_t x, const int16_t y, const uint16_t width, const uint16_t height, const uint8_t cornerRadius, const uint32_t color);
 
+    // KTOME function for triangles
+    /*!
+        @brief  Draw triangle in current display list. Points must be defined in clockwise order. Only tested with equilateral tris.
+        @param  x1 x-coordinate for the first triangle point
+        @param  y1 y-coordinate for the first triangle point
+        @param  x2 x-coordinate for the first triangle point
+        @param  y2 y-coordinate for the first triangle point
+        @param  x3 x-coordinate for the first triangle point
+        @param  y3 y-coordinate for the first triangle point
+        @param  color Color of the rectangle
+        @param  bgcolor Color of the background
+    */
+    void drawTri(const int16_t x1, const int16_t y1, const int16_t x2, const int16_t y2, const int16_t x3, const int16_t y3, const uint32_t color,const uint32_t bgcolor);
```

Comment:
I got the linter up and running again and it's complaining about this missing space 😉 

```suggestion
    void drawTri(const int16_t x1, const int16_t y1, const int16_t x2, const int16_t y2, const int16_t x3, const int16_t y3, const uint32_t color, const uint32_t bgcolor);
```

Reply:
Hey is this something I need to fix before the merge can take place?

Reply:
Yeah, I'd like to keep the code formatted consistently, that's why I automatically run a linter on it for each pull request. You can just click _commit suggestion_, though. There may be other issues coming up afterwards, let's see 😅